### ISSUE TITLE
TypeError: 'numpy.ndarray' object is not callable matlab2python

### ISSUE THREAD

**Issue by SDIdo** [ebranlard/matlab2python#4]
Dear ebranlard,
Thank you for making this amazing tool.
We have almost everything converted out-of-the-box with minor bearable changes.

Issue:
The recognizing of a matlab array or matrix is needed many times.
Matlab2Python, currently, doesn't recognize and thus takes slicing as a function call.
This probably makes it not recognize the index change that is needed as well.

Recreation:

test.m
```
x = [1:4,11:14];
y = x(1,1);
disp(y);
```

test.py
```
import numpy as np
x = np.array([np.arange(1,4+1),np.arange(11,14+1)])
y = x(1,1)
print(y)
```

y = x(1,1), at run, will cause the following error
`TypeError: 'numpy.ndarray' object is not callable `

Will be glad for a fix or a direction towards what to change in the code.

Best Regards,
SDido

**Comment by ebranlard** [comment]
Hi @SDIdo, 

Sorry for the late reply. Thanks for reporting this and for your nice and simple example. 

Those are indeed hard to parse... So far I've updated them manually. 
I've been a bit away from this project for a while, so I wouldn't have any recommendations as to how to go about this. I would have to dive into the code, and possibly into the SMOP code. Your test case is definitely a good start.
